platform/broadcom/saibcm-modules/sdklt/linux/bde/ngbde_dma.c

```diff
@@ -224,7 +227,9 @@ ngbde_dmamem_free(ngbde_dmamem_t *dmamem)
         }
         dma_free_coherent(dmamem->dev, dmamem->size,
                           dmamem->vaddr, dmamem->paddr);
-        memset(dmamem, 0, sizeof(*dmamem));
+        #ifdef __STDC_LIB_EXT1__
+            memset_s(*dmamem, sizeof(*dmamem), 0, sizeof(*dmamem));
+	#endif
```

Comment:
>	 [](http://example.com/codeflow?start=0&length=1)

Do not mix using tabs and spaces for indentation. Seems this file is using 4-space as indentation.

Reply:
fixed.